src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random;
+
+    public ColorSupplier() {
+        random = new Random();
+    }
+
+    public String getRandomColor() {
+        Color[] colors = Color.values();
+        int randomNumber = random.nextInt(colors.length);
+        return colors[randomNumber].toString();
```

Comment:
```suggestion
        return colors[randomNumber].name();
```
![image](https://github.com/mate-academy/jv-oop-advanced/assets/49767049/919b0768-6f22-41bd-a871-31c4ad4bed11)

Reply:
not fixed